packages/syntax-core/src/Chip/Chip.tsx

```diff
@@ -123,7 +123,7 @@ const Chip = forwardRef<HTMLButtonElement, ChipProps>(
         aria-pressed={selected}
         onClick={onChange}
       >
-        {Icon && <Icon className={iconStyles} size="sm" />}
+        {Icon && <Icon className={iconStyles} size={100} />}
         <Box paddingX={Icon ? 1 : 0}>
```

Comment:
@christianvuerings @somethiiing The issue I'm getting with these other components that use `Icon` like `Chip`, `IconButton` etc is that we are still using both Material icons and our own Icon.

Now with this change, the sizing of them is different. Material uses `sm` `med` `lg` and our Icon component will now use `100` `200` etc. 

How do I determine what type of Icon is being passed in as a prop?

Reply:
cant you just pass in both props?

Reply:
They both use the name `size` as the prop.

I think I got it to work though. The `size` prop that's passed down controls the size for our internal icons and for material the className that was being used controls the size of the icon